### ISSUE TITLE
helpers no longer receive env argument in ember 1.13

### ISSUE THREAD

**Issue by jpaas** [wildland/ember-sanctify#10]
The can-access helper is broken in ember 1.13 because helpers no longer receive the env argument. This causes `Uncaught TypeError: Cannot read property 'view' of undefined` at https://github.com/wildland/ember-sanctify/blob/master/addon/helpers/can-access.js#L69

**Comment by jpaas** [comment]
ember-can had the same problem and discussed it with the ember team https://github.com/emberjs/ember.js/issues/11080

**Comment by Samsinite** [comment]
@jpaas I have a branch at https://github.com/wildland/ember-sanctify/tree/update-to-1.13 that I have already updated to hopefully work with 1.13. Haven't tested it in an app yet and not sure exactly how to test the new `Ember.Helper` object. If you'd like to give it a try and provide feedback that would be very helpful.

_Edit:_ Looks like `ember-can` just punted on unit testing the helper inside of their addon.

**Comment by Samsinite** [comment]
Looks like ember-can just punted on unit testing the Ember.Helper object in their lib, I might do the same.

**Comment by jpaas** [comment]
I gave that branch a try and I found that the route canAccess policies work, but my can-access helpers are not working. Pretty much all of my can-access helpers use the canAccess method on my route policies like so..

```
{{#if (can-access 'authenticated/resources/new' 'access')}}
...
{{/if}}
```

If I set a break point in the policy, its being called and appears to return true, but the conditional block is not being executed.

**Comment by Samsinite** [comment]
Thanks for the update. I'll take a look at this later tonight and try to get a fix in by tomorrow along with an official release. If you would like to take a stab at it sooner, feel free.

*Edit: \* Also, the next release will change the resolver lookup from the folder `app/policy/...` to `app/policies/...`, though that hasn't been merged from master into the branch above yet.

**Comment by Samsinite** [comment]
@jpass Are you able to tell if the `recomputeAccess` observer is firing after the promise resolves and changes the `canAccess` property?

**Comment by jpaas** [comment]
Sorry, I had to keep moving forward and disabled sanctify for now

**Comment by Samsinite** [comment]
This has been resolved in #11 and 0.2.0 has been published. You'll need to rename your `app/policy` directory to `app/policies`
